go/vt/vttablet/endtoend/views_test.go

```diff
@@ -41,16 +42,38 @@ func TestCreateViewDDL(t *testing.T) {
 		&vtrpcpb.CallerID{},
 		&querypb.VTGateCallerID{Username: "dev"}))
 
-	defer client.Execute("drop view vitess_view", nil)
+	ch := make(chan any)
+	ctx, cancel := context.WithCancel(context.Background())
+	defer cancel()
+	go func() {
+		err := client.StreamHealthWithContext(ctx, func(shr *querypb.StreamHealthResponse) error {
+			views := shr.RealtimeStats.ViewSchemaChanged
+			if len(views) == 0 {
+				return nil
+			}
+			if views[0] == "vitess_view" {
+				ch <- true
+			}
+			return nil
```

Comment:
nit picking: this can be simplified as below:

```suggestion
			if len(views) != 0 && views[0] == "vitess_view" {
				ch <- true
			}
			return nil
```